### ISSUE TITLE
Track lines when parsing copyright files

### ISSUE THREAD

**Issue by pombredanne** [nexB/debian-inspector#13]
This would be helpful when reviewing results

**Comment by pombredanne** [comment]
I have started to work on this as the lack of proper start and end line in Debian copyright license detection  makes reviewing test failures rather hard.

I searched for a library that track line numbers and could not find any. And none seems really suitable to patch and add line numbers.

So my overall approach is going to be:
- add a new module that can parse RFC822/deb822 formats and track lines for each elements at the low level (essentially replacing our use of the standard `email`module)
- create a new `debcon2.py` module modeled after `debcon.py` that will track line numbers at the paragraphs and field levels
- create a new `copyright2.py` module modeled after `copyright.py` that will track line numbers at the paragraphs and field levels and is backed by `debcon2.py`
- add tests
- try copyright2.py in ScanCode for #2643
- See if debcon2 and copyright2 can be dumbed down and also not track licenses when not needed and replace the older debcon and copyright. Otherwise, keep the debcon and copyright around with deprecation warnings and drop them in the future.
